README.md

```diff
@@ -8,7 +8,15 @@
 ## Working Changes
 [Read the working changes to the spec](spec.md)
 
-These changes have been accepted by the working group for the next version of the specification, however are still subject to change.
+These changes have been accepted by the working group for the next version of
+the specification, however are still subject to change.
+
+## Alpha Features
```

Comment:
I agree, I learned it was double-spaces after a period too, but I guess html redefined that for us  :-)